CONTRIBUTING.md

```diff
@@ -55,6 +55,14 @@ git checkout -b cool-new-feature
 
 4\. When you are ready, submit a pull request into the composer repository! If merged, we'll reach out to send you some free swag :)
 
+## Pull request guidelines
+
+We have some rough guidelines that will make your PR easier to review and more likely to get smoothly merged.
+* Self-contained title and description. Please include a descriptive title and description. These should allow someone to understand the contents of the PR without looking at the code.
```

Comment:
The title should be descriptive yet concise.  Ideally the title should answer the question, "what does this PR change or do?".  There should be enough information that users can understand at a glance and if they're interested can click to read additional details.

For the PR description I'd like to emphasize the bit that user's (technical and possibly semi-technical) can understand what a given PR does _without_ looking at the code.  In plain English, the description should highlight what the PR changes and what the impact to user's will be (if any).